canonical-web-and-design/vanilla-framework
# Add individual examples for content patterns

## Done

Makes sure "content" related patterns can be built separately and add individual examples for them.

Fixes #2750 

## QA

NOTE: Visual side of the examples listed below was verified by CI on Percy, so QA steps are to make sure examples work as expected and detailed visual review of those is not needed.

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/ or [demo](https://vanilla-framework-canonical-web-and-design-pr-2762.run.demo.haus/)
- Go to [examples page](https://vanilla-framework-canonical-web-and-design-pr-2762.run.demo.haus/)
- Make sure that existing examples (full Vanilla) for components below work as before (check different variants for each pattern):
buttons
heading icon
headings
image
inline images
muted heading
links
label
notifications
pull quote
tooltips
- Make sure that individual examples for these components work the same as full ones
